week2/MAKEME.md

```diff
@@ -125,9 +123,9 @@ Next up:
 
 **1.3 Deleting posts**
 
-To delete a post we need to target a file by using an identifier: the title. However, instead of getting the title through the body of the request, we're going to get it from the `URL parameters`.
+For deleting posts we will again make use of `URL parameters`, this time to specify which post we want to delete.
 
-Since we are deleting a file there is no need to send any content in the request. To delete a file with `fs`, we'll use the `fs.unlinkSync(<filename>)` method.
+Since we are deleting a post there is no need to send any content in the request. To delete the corresponding, you can use `fs.unlinkSync(<filename>)`.
```

Comment:
```suggestion
Since we are deleting a post there is no need to send any content in the request. To delete the corresponding file, you can use `fs.unlinkSync(<filename>)`.
```